src/app/globals.css

```
@@ -262,6 +262,83 @@ body {
+  width: 100%;
+  min-width: 100%; /* 추가 */
+}
+
+.ProseMirror table colgroup col {
+  min-width: 40px; /* 열이 너무 좁아지는 것 방지 */
+}
+
+.ProseMirror table td,
+.ProseMirror table th {
+  overflow: hidden;
+  word-break: break-word;
```

Stylelint (17.3.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)